### PR TITLE
Cortex XDR - script executions functionality

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/31185

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@anara123 @ShahafBenYakir @yaakovi Doc review completed.

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 1]
```diff
@@ -1077,7 +1077,10 @@ def get_script_code(self, script_uid) -> Dict[str, Any]:
 
         return reply.get('reply')
 
-    def run_script(self, script_uid, endpoint_ids: list, timeout: int, parameters: Dict[str, Any]) -> Dict[str, Any]:
+    @logger
+    def run_script(self,
```
Comment (yaakovi) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:1081]: why we had this function if it wasn't used?

Comment (Itay4) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:1081]: was implemented before api was ready

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 2]
```diff
@@ -2736,6 +2787,157 @@ def action_status_get_command(client: Client, args) -> Tuple[str, Any, Any]:
...
+        response
+    )
+
+
+def run_script_kill_process_command(client: Client, args: Dict) -> Tuple[str, Any, Any]:
+    endpoint_ids = argToList(args.get('endpoint_ids'))
+    try:
+        timeout = int(args.get('timeout', 600))
+    except ValueError:
+        raise ValueError('The timeout argument need to be an integer.')
```
[the remark this answers is not in the document]
Comment (yaakovi) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py]: no reason to wrap it in try-except, the raised exception in the function is fine

[the remark this answers is not in the document]
Comment (Itay4) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py]: done

**Review by yaakovi — APPROVED** [review]
🍪

**Pull request merged**